### PR TITLE
Add EmployeeSalaryKey and EmployeeAlternateID

### PR DESCRIPTION
**Pull request by xhocquet**

### CODE REVIEW

**Comment by xhocquet** [comment]
@puneet-sutar Could you please review this? Seemed pretty straight-forward

File: lib/equifax/worknumber/voe/instant.rb [hunk 1]
```diff
@@ -23,6 +23,8 @@ def xml
               <REQUEST InternalAccountIdentifier="#{account_number}"
                 LoginAccountIdentifier="#{account_number}" LoginAccountPassword="#{password}">
                 <KEY _Name="EMSEmployerCode" _Value="#{employer_code}" />
+                <KEY _Name="AlternateID" _Value="#{employee_alternate_id}" />
+                <KEY _Name="EMSSALARYKEY" _Value="#{employee_salary_key}" />
```
Comment (puneet-sutar) [inline lib/equifax/worknumber/voe/instant.rb]: It might be better to not add the tags if the value is blank

**Comment by xhocquet** [comment]
@puneet-sutar No, this is only available on Instant based on their documentation

File: lib/equifax/worknumber/voe/instant.rb (hunk 1 above)
Comment (xhocquet) [reply lib/equifax/worknumber/voe/instant.rb]: `employer_code` above is also an optional field, but appended regardless. Was just following that pattern, but I agree it's weird to send it unnecessarily

File: lib/equifax/worknumber/voe/instant.rb [hunk 2]
```diff
@@ -23,6 +23,8 @@ def xml
               <REQUEST InternalAccountIdentifier="#{account_number}"
                 LoginAccountIdentifier="#{account_number}" LoginAccountPassword="#{password}">
                 <KEY _Name="EMSEmployerCode" _Value="#{employer_code}" />
+                <KEY _Name="AlternateID" _Value="#{employee_alternate_id}" />
```
Comment (puneet-sutar) [inline lib/equifax/worknumber/voe/instant.rb]: @xhocquet You are not using the functions here

Comment (xhocquet) [reply lib/equifax/worknumber/voe/instant.rb]: Good catch! :swea

**Pull request merged**